_posts/2014-08-18-nutella-baguette.md

```diff
@@ -1,3 +1,4 @@
+Add some chestnut cream
```

Comment:
This line should come in the end @audedieude :smile:

If you notice all files start with some metadata that is used to categorize the recipe. Adding this line at the very top will break that, so just do another commit removing this line and adding it to the ingredients section :+1: :smile: